app/controllers/project.js

```diff
@@ -0,0 +1,63 @@
+const Project = require('../models/Project')
+
+module.exports = {
+  createProject: async (req, res, next) => {
+    const project = new Project(req.body)
+    try {
+      await project.save()
+      res.send(project)
+    } catch (error) {
+      console.log(error)
```

Comment:
@JanviMahajan14 console.log only in env other than production environment.
So add if(process.env.NODE_ENV !== 'production'){ conso..... }